examples/expositional/end2end_apps/trubot/trubot.py

```diff
@@ -3,31 +3,30 @@
 from pprint import PrettyPrinter
 from typing import Dict, Set, Tuple
 
-from langchain.chains import ConversationalRetrievalChain
-from langchain.embeddings.openai import OpenAIEmbeddings
-from langchain.memory import ConversationSummaryBufferMemory
 from langchain_community.llms import OpenAI
 from langchain_community.vectorstores import Pinecone
 import numpy as np
 import openai
 import pinecone
 from slack_bolt import App
 from slack_sdk import WebClient
+from trulens.core import Feedback
+from trulens.core import Select
+from trulens.core import Tru
+from trulens.core.schema.feedback import FeedbackMode
+from trulens.langchain import TruChain
```

Comment:
Let's put lanchain, llama-index-etc in a category. Something like trulens.frameworks.langchain, trulens.frameworks.llama_index.

This will more clearly distinguish from langchain feedback provider, etc.